certified-connectors/CobbleStone - Contract Insight/apiDefinition.swagger.json

```diff
@@ -0,0 +1,965 @@
+          },
+          {
+            "name": "query",
+            "in": "body",
+            "description": "Query Model",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/APIQueryRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "$ref": "#/definitions/APITriggerQueryResponse"
+            }
+          }
+        }
+      }
+    },
+    "/CSSAPI/v2/PowerAutomate/PollTriggers/{entityName}/NewAndUpdated": {
+      "post": {
+        "tags": [
+          "Poll Triggers"
+        ],
+        "summary": "A scheduled trigger for created and updated records",
+        "description": "A scheduled trigger that shows both created (new) records and any records which have been updated since a given amount of time.",
+        "operationId": "CSSAPI_NewAndUpdated_Records_Poll_Trigger",
+        "x-ms-trigger": "batch",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-trigger-metadata": {
+          "mode": "polling",
+          "kind": "query"
+        },
+        "parameters": [
+          {
+            "name": "entityName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Table Name",
+            "x-ms-url-encoding": "single",
+            "description": "Table name to pull updated record(s) from.",
+            "x-ms-dynamic-values": {
+              "operationId": "CSSAPI_EntityListUpdate",
+              "value-path": "EntityNameForAPI",
+              "value-title": "EntityNameDisplay"
+            }
+          },
+          {
+            "name": "since",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "format": "date-time",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "query",
+            "in": "body",
+            "description": "Query Model",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/APIQueryRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "$ref": "#/definitions/APITriggerQueryResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "APIEntityListResponse": {
+      "description": "List of Entities",
+      "type": "object",
+      "properties": {
+        "EntityName": {
+          "description": "Entity name as it is in database",
+          "type": "string"
+        },
+        "EntityNameDisplay": {
+          "description": "Entity display name",
+          "type": "string"
+        },
+        "AllowAdd": {
+          "description": "Indicates if allowed to add new record",
```

Comment:
@nizarhandal please make sure everything is grammatically correct and free of typographical errors, refer to this [guideline](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).